### ISSUE TITLE
[AMANDA] Update Outgoing Email Functionality, Comments and Format for ROW Folders (SS) (On PROD)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#11025]
**Acceptance Criteria**

- The Acceptance Criteria provided for this ticket applies to RW, DS, EX and PSM folders. 
- The recipients of all emails need to be updated/corrected and the emails should ONLY be sent to the following people types
-- Applicant
-- Right of Way Contractor (if listed)
-- Contact (if listed)
-- Billed To (if listed)
- Provided below are the scenarios through which the emails are generated
-- By making attempts Department Comments, Accepted and Rejected in the Web Acceptance Process
-- By clicking on Options-> Email Bill from a generated bill in the Fee tab. 
-- By clicking on Options-> Print -> Email/Print Permit from the Folder tab.

> What application are you using?

AMANDA

> Describe the problem.

All emails sent on Right of Way folders, via AMANDA, to be sent to specific parties.  We'll have to generate this list as a group.  For now, we want them to send to:
- Applicant
- Right of Way Contractor (if listed)
- Contact (if listed)
- Billed To (if listed)

And we do NOT want them to send to:
- "Previous Responsible Party"
- "Previous Billed To"
- "Inactive Contractor"
- "Owner" (I think?)

> Describe the outcome you'd like to see when this feature is implemented.

Emails such as Department Comments, Invoices, and Permits will be sent to appropriate parties and NOT to parties no longer associated w/ the permit.

The trick is

> Describe any workarounds you currently have in place or alternative solutions you've considered.

For the Department Comments emails, there is no workaround.
For the Invoice emails, we can remove incorrect email addresses.  
For the Permit emails, we can remove incorrect email addresses.  

The two workarounds noted above are not ideal since it requires the Analyst to remember to make this change manually.

> Is there anything else we should know?

The sooner the better!

> Requested By
Paloma A.

Request ID: DTS22-105895

**Comment by TracyLinder** [comment]
ROW asked for an update. We'll need to get this to scoping soon.
FYI @Nadin-Nader and @vschandramouli

**Comment by Nadin-Nader** [comment]
**Update:) Subha is Analyzing

**Comment by SubhaShroff** [comment]
@Nadin-Nader 
WAA processes changes for DS,EX,PSM,RW is done and I will be sending it for DEV deployment. 
Now I have started with the reports. 
Thanks.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0297758 is submitted for DEV promotion. Thanks.

**Comment by Nadin-Nader** [comment]
@SubhaShroff here is Paloma's response:

[11:31 AM] Amayo-Ryan, Paloma
    Nader, Nadin - that shouldn't be an issue since an Applicant wouldn't be able to apply for a permit w/o a valid Internet User record.  The other records - Billed To, Right of Way Contractor, and Contact - do NOT/should NOT need to be associated to any other records nor should they need to be validated.

Reviewing the Acceptance Criteria for this ticket and it looks solid to me.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder I got to talk to Marco Pasin(Granicus), we don't have a solution to restrict from Folder Tab, Fee tab.

**Comment by Nadin-Nader** [comment]
Per our Update: There has been an update  to where this may be possible. Subha is looking in to it

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0300897 is submitted for DEV deployment of processes mending.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0302013 is submitted for the manual TEST deployment for the above listed reports.

**Comment by SubhaShroff** [comment]
@Nadin-Nader 
DEV tested, following are the results.
1. DS - 12532726 - Working as expected.
2. EX - 12532472 - working as expected
3. RW - 12532567 - working as expected.
4. PSM -  Testing............

**Comment by SubhaShroff** [comment]
@Nadin-Nader  After you have approved in the stand up for TEST deployment, I have raised SCTASK0302513 request, for deploying the Commits in TEST.
SCTASK0302013 - Manual TEST request
SCTASK0302513 - commits TEST request

**Comment by SubhaShroff** [comment]
@Nadin-Nader  TEST deployment is done. And I have tested for the following and its ready for UAT  on TEST.
DS - rsn-12877238 
PSM - 12876929
EX - 12877321
RW - 12877321
ssBilledTo@yopmail.com
ssContact@yopmail.com
SSRowContractorTEST@yopmail.com
justanyPeople@yopmail.com

**Comment by Nadin-Nader** [comment]
@SubhaShroff Kyle has **approved** the move to PROD. Please send to CTM for PROD Deployment

**Comment by SubhaShroff** [comment]
@Nadin-Nader  

1. SCTASK0310955 Is submitted for PROD deployment.
2. SCTASK0310958 - is submitted for Manual PROD deployment.

Thanks

**Comment by Nadin-Nader** [comment]
Sent Message to Kyle and ROW Team to please UAT and let us know if this is good to close or not now that it is on PROD. Waiting on Response

**Comment by Nadin-Nader** [comment]
This has been approved on PROD by ROW, Closing

**Comment by TracyLinder** [comment]
### Update

- Kim has an example of a DS where the Web Acceptance email didn't go to the applicant, it only went to the contact. Email was sent 6/15, the folder RSN is 13146062.

**Comment by KyleErich** [comment]
Needs to be reopened for additional scoping: An issue was identified on FolderRSN 13211339 in which the ROW Contractor received the Department Comments email but the Applicant did not.

**Comment by vschandramouli** [comment]
Moved to Backlog since this needs to be taken up for bug fixing. Issue reported by Kyle seems similar to #12663

**Comment by TracyLinder** [comment]
Issue #14081 was created for the recent bug instead of reopening this one.
